runelite-api/src/main/java/net/runelite/api/Skill.java

```diff
@@ -58,6 +60,16 @@
 	 */
 	OVERALL("Overall");
 
+	private static final Map<String, Skill> skillMap = new HashMap<String, Skill>()
```

Comment:
Use a `static { }` block to initialize the map instead

Reply:
`Enum.valueOf` would work too